src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -133,7 +136,9 @@
         },
         watch: {
             uploadFiles() {
-                this.setDefaultCheckedItems()
+                if (!this.defaultSet){
```

Comment:
If you really do want to select the zip and html files only when they are first added then it would be cleaner to do something like this, avoiding need for `defaultSet` variable:
```kotlin
uploadFiles(newValue, oldValue) {
    if (!oldValue) {
        this.setDefaultCheckedItems()
    }
}
```
Alternatively you could make setDefaultCheckedItems re-runnable... something like this:
```kotlin
setDefaultCheckedItems() {
    this.uploadFilesToAdr = ["outputZip", "outputSummary"].filter(key => this.uploadFiles.hasOwnProperty(key))
}
```